_layouts/tutorial_hands_on.html

```diff
@@ -32,22 +32,30 @@
           <div class="collapse navbar-collapse" id="navbar">
 
             <ul id="main-menu" class="nav navbar-nav navbar-right">
-              <li><a href="{{ site.url }}"><i class="fa fa-home"></i> Home</a></li>
-              <li><a href="{{ site.url }}/{{ topic.name }}"> {{ topic.name }}</a></li>
-              <li><a href="{{ site.url }}/{{ topic.name }}/slides"><i class="fa fa-graduation-cap"></i> Introduction slides</a></li>
+              <li><a href="{{ site.url }}" title="Back to front page"><i class="fa fa-home"></i> Home</a></li>
+              <li><a href="{{ site.url }}/{{ topic.name }}" title="Go back to list of tutorials"><i class="fa fa-file-text-o"></i> {{ topic.name }}</a></li>
+
+              {% if tutorial.slides != "" %}
+                <li><a href="{{ site.url }}/{{ topic.name }}/slides" title="Slides for this tutorial"><i class="fa fa-graduation-cap"></i> Introduction slides</a></li>
+              {% endif %}
               <!--<li><a href="{{ site.url }}/{{ topic.name }}/slides/{{ tutorial.name }}/"><i class="fa fa-file-powerpoint-o"></i> Associated slides</a></li>-->
 
-              {% if topic.docker_image != "" %}
```

Comment:
As @bgruening suggested it probably only makes sense to have topic-level docker images, so I'll rethink that